Basket-Donation-Payments/Basket-Donation-Payments/View Controllers/Login & Signup Flow/InitialLoginViewController.m

```diff
@@ -53,7 +53,18 @@ - (IBAction)loginButtonTapped:(id)sender {
 
         [self presentViewController:alert animated:YES completion:nil];
     } else {
-        [self performSegueWithIdentifier:@"loginSegue" sender:nil];
+        UIActivityIndicatorView *loadingIndicator = [Utils createUIActivityIndicatorViewOnView:self.view];
```

Comment:
Same here, you're leaking memory if you don't remove any existing loadingIndicator from the view before adding a new one.